### DISCUSSION TITLE
pangyo-evening-meet-up-http-241023/

### DISCUSSION THREAD

**Discussion by giscus** [devmeeple/devmeeple.github.io#14] [Comments]
# pangyo-evening-meet-up-http-241023/

10월 23일, 약 300명의 경쟁률을 뚫고(?) 퇴근길 밋업에 다녀왔다. 밋업은 발표 세션과 질의 응답, 네트워킹 순서로 진행됐다. 발표 세션 : 웹의 시작 (HTTP의 탄생과 기초를 쉽게 이해하는 시간) 강의에 내용 맛보기를 제공해 주셨다. 특별한 점은, 실습과 함께 요청, 응답 흐름을 이야기해 주셨다.
HTTP 0.9를 기준으로 간단한 코드를 제공해…

https://devmeeple.github.io/pangyo-evening-meet-up-http-241023/

**Comment by SungbinYang** [comment]
원따봉 누르고 갑니다!

**Reply by devmeeple** [reply]
항상 관심 가져주셔서 감사합니다. 성빈 님 :)

**Comment by yook9688** [comment]
따봉 추

**Reply by devmeeple** [reply]
와.정.말.감.사.합.니.다.
